scala/lambdas/utils/src/main/scala/uk/gov/nationalarchives/utils/ExternalUtils.scala

```diff
@@ -112,6 +112,10 @@ object ExternalUtils {
           assetMetadataIdFields
         ) =>
       val convertListOfUuidsToJsonStrArray = (fileUuids: List[UUID]) => fileUuids.map(fileUuid => Json.fromString(fileUuid.toString))
+      val upstreamSystemValue = upstreamSystem match {
+        case SourceSystem.PA => "Parliament Migration"
```

Comment:
Should we save "Parliament Migration" to some sort of variable or as a property of the enum itself?

Reply:
I think it's ok as is.